apps/taglaunch/app.js

```diff
@@ -106,8 +106,9 @@ let showTagMenu = (tag) => {
       let app = appsByTag[tag][i];
       if (!app) return;
       if (!app.src || require("Storage").read(app.src)===undefined) {
+        Bangle.setUI();
```

Comment:
Otherwise, you can still scroll and click the underlying buttons while the warning is displayed. IMO that is pretty buggy and not wanted.

Reply:
I use `setUI` because of this comment: https://github.com/espruino/BangleApps/pull/3319#issuecomment-2031744595

I am not exactly sure what the benefits and tradeoffs are here, but from my previous testing it did not seem to interfere with the widgets.